mongodbatlas/resource_mongodbatlas_federated_database_instance.go

```diff
@@ -239,10 +240,6 @@ func schemaFederatedDatabaseInstanceStores() *schema.Schema {
 					Type:     schema.TypeString,
 					Optional: true,
 				},
-				"cluster_id": {
```

Comment:
Removing cluster_id as this has been deprecated and not available in the new SDK anymore.

Reply:
cc @Zuhairahmed